test/unit/configuration/configuration_parser_test.js

```diff
@@ -0,0 +1,139 @@
+const assert = require('assert');
+const configurationParser = require('./../../../lib/configuration/configuration_parser');
+const fsPromises = require('fs/promises');
+const os = require('os');
+const path = require('path');
+const { v4: uuidv4 } = require('uuid');
+const tempDir = path.join(os.tmpdir(), uuidv4());
+
+describe('Configuration parsing tests', function () {
+
+  before(async function () {
+    await fsPromises.mkdir(tempDir, { recursive: false });
+  });
+
+  after(async function () {
+    await fsPromises.rm(tempDir, { recursive: true, force: true });
+  });
+
+  it('should parse json', async function () {
+    // given
+    const fileName = 'config.json';
+    const filePath = path.join(tempDir, fileName);
+    const fileContent = `{
+            "common": {
+                "log_level": "info",
+                "log_path": "/some-path/some-directory"
+            } 
+        }`;
+    await fsPromises.writeFile(filePath, fileContent, { encoding: 'utf8' });
+
+    // when
+    const configuration = await configurationParser.parse(filePath);
+
+    // then
+    assert.equal(configuration.logLevel, 'info');
+    assert.equal(configuration.logPath, '/some-path/some-directory');
```

Comment:
let's extract common variables used in the test

Reply:
done